jobs/sync-anime/sync-anime.go

```diff
@@ -58,7 +58,13 @@ func sync(data *kitsu.Anime) *arn.Anime {
 	anime.EndDate = attr.EndDate
 	anime.EpisodeCount = attr.EpisodeCount
 	anime.EpisodeLength = attr.EpisodeLength
-	anime.Status = attr.Status
+
+	if strings.EqualFold(attr.Status, "tba")  || strings.EqualFold(attr.Status, "unreleased") {
+		anime.Status = "Upcoming"
+	}  else {
+		anime.Status = attr.Status
```

Comment:
I would remove the else block and always assign to `anime.Status` before the `if`. Then simply check `anime.Status` in the if condition without an else block.

Reply:
I usually do it that way but sometimes that confuses people apparently because they forget that has a default clause before the `if` so I wasn't sure if you wanted me to use and `else` or just have a default clause before the `if` statement.